tsdb/wal/watcher.go

```diff
@@ -149,7 +151,8 @@ func NewWatcher(metrics *WatcherMetrics, readerMetrics *LiveReaderMetrics, logge
 		quit:          make(chan struct{}),
 		done:          make(chan struct{}),
 
-		MaxSegment: -1,
+		MaxSegment:  -1,
```

Comment:
the comment for `MaxSegment` suggests it is supposed to be used only in tests, we should confirm how it's being used right now and either a) change the comment or b) create a new variable to use instead of `MaxSegment`

Reply:
i think this was already there before.

Reply:
Yeah it was. I thought I saw other code using `MaxSegment` but maybe I was mistaken, it looks like you're using `maxSegmentNumber` now. This change (extra space) should be removed though, as it's just noise when someone is trying to review your PR :)